### PR TITLE
Upload jars to bintray

### PR DESCRIPTION
**Pull request by lguo**

### CODE REVIEW

File: jcenter/bintray.gradle [hunk 1]
```diff
@@ -0,0 +1,108 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
```
Comment (yunboouyang) [inline jcenter/bintray.gradle:2]: Should it be 2020?

File: jcenter/maven-install.gradle [hunk 2]
```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
```
Comment (yunboouyang) [inline jcenter/maven-install.gradle:2]: same here

File: jcenter/bintray.gradle [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*
+MIT License
```
Comment (yunboouyang) [inline jcenter/bintray.gradle:17]: Shall we unify the comment styles for Apache / MIT licenses?

File: release-bintray.gradle [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  artifact = POM_ARTIFACT_ID
+  packagingType = POM_PACKAGING
+  libraryDescription = 'Photon Machine Learning (Photon ML)'
+
+  siteUrl = 'https://github.com/linkedin/photon-ml'
+  gitUrl = 'https://github.com/linkedin/photon-ml.git'
+
+  libraryVersion = '19.0.0'
+
+  licenseName = 'Apache-2.0'
```
Comment (junshi15) [inline release-bintray.gradle:61]: Is this the original license? Linkedin has moved to BSD-2 license. but we don't need to change here, I suppose. https://github.com/linkedin/detext/blob/master/LICENSE.md

File: jcenter/bintray.gradle (hunk 1 above)
Comment (lguo) [reply jcenter/bintray.gradle:2]: fixed.

File: jcenter/maven-install.gradle (hunk 2 above)
Comment (lguo) [reply jcenter/maven-install.gradle:2]: fixed.

File: jcenter/bintray.gradle (hunk 3 above)
Comment (lguo) [reply jcenter/bintray.gradle:17]: fixed.

File: release-bintray.gradle (hunk 4 above)
Comment (lguo) [reply release-bintray.gradle:61]: We have decided to stick to the original photon-ml license, Apache 2.0.

**Comment by junshi15** [comment]
There was a unit test failure. Please take a look.

**Pull request merged**